src/js/components/layout.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { HashRouter as Router, Route, Switch } from 'react-router-dom';
+import { createBrowserHistory } from 'history';
+import Search from './Search.jsx';
+import Login from './Login.jsx';
+import LoginMessage from './LoginMessage.jsx';
+import Article from './Article.jsx';
+import Footer from './Footer.jsx';
+import Sources from './sources.jsx';
+import NotFound from './NotFound.jsx';
+
+const history = createBrowserHistory();
+class Layout extends React.Component {
+  constructor() {
+    super();
+    this.state = {};
+  }
```

Comment:
you can remove Lines 14 to 17 from here as it does nothing really. Also, express the component as a functional component. refer to comment in `src/js/components/Footer.jsx` above.